src/otx/core/utils/mask_util.py

```diff
@@ -8,13 +8,19 @@
 from typing import TYPE_CHECKING
 
 import pycocotools.mask as mask_utils
+import torch
 
 if TYPE_CHECKING:
     import numpy as np
     from datumaro import Polygon
 
 
-def polygon_to_bitmap(polygons: list[Polygon], height: int, width: int) -> np.ndarray:
+def polygon_to_bitmap(
+    polygons: list[Polygon],
+    height: int,
+    width: int,
+    return_rle: bool = False,
+) -> np.ndarray:
```

Comment:
I guess that this can be separated into two functions for clarity and performance
```suggestion
def polygon_to_rle(
    polygons: list[Polygon],
    height: int,
    width: int
) -> np.ndarray:
    polygons = [polygon.points for polygon in polygons]
    return mask_utils.frPyObjects(polygons, height, width)

def polygon_to_bitmap(
    polygons: list[Polygon],
    height: int,
    width: int,
    return_rle: bool = False,
) -> np.ndarray:
    canvas = np.zeros([height, width], dtype=np.uint8)
    return drawing_masks(canvas, polygons)
```

Reply:
Thanks for the comment. I added the suggestion accordingly.